andela/ah-frontend-invictus
# #166324467 (chore): set up eslint and add pre-commit hooks

#### What does this PR do?
- adds eslintrc.json file
- adds husky to enforce pre-commit hooks

#### Description of Task to be completed
- When the git commit command is used, the code is first linted and if there any errors process aborts.

#### How should this be manually tested?
- checkout to this branch and use `npm install`
- make a commit and notice the code is linted first

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#166324467 Add pre-commit hooks to lint code](https://www.pivotaltracker.com/story/show/166324467)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28973383/58549861-25b42d80-8215-11e9-824d-e74d6b470167.png)
